staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go

```diff
@@ -33,6 +33,10 @@ import (
 	"k8s.io/client-go/dynamic"
 )
 
+const (
```

Comment:
why this change part of this PR?

Reply:
otherwise arm 32 bit fails due to that the constant is larger than int32 (default on arm) can handle

Reply:
okay